eulerpi/core/inference.py

```diff
@@ -60,6 +60,77 @@ def inference(
         Tuple[Dict[str, np.ndarray], Dict[str, np.ndarray], Dict[str, np.ndarray], ResultManager]: The parameter samples, the corresponding simulation results, the corresponding density
         evaluations for each slice and the result manager used for the inference.
 
+
```

Comment:
Could we insert the `examples:` tag everywhere? This will give us a small heading for the examples and also transfer to other doc styles nicely if we ever switch.